src/index.html

```diff
@@ -13,5 +13,188 @@
+          <label>
+         Full date of birth:
+            <input type="date"
+         name="date"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+         I accept the term of the agreement
+            <input type="checkbox"
+         name="agreement"
+            >
+          </label>
+        </div>
+
+      </fieldset>
+
+      <fieldset class="set">
+        <legend>
+          Registration
+        </legend>
+
+        <div class="form-field">
+          <label>
+          E-mail:
+            <input type="email"
+          name="email"
+          placeholder="email@example.com"
+          required
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+          Password:
+            <input type="password"
+          name="password"
+          required
+          minlength="8"
+          maxlength="20"
+            >
+          </label>
+        </div>
+
+      </fieldset>
+
+      <fieldset class="set">
+        <legend>
+          An interesting fact about you!
+        </legend>
+
+        <div class="form-field">
+          <label>
+          Do you love cats?
+            <input type="radio"
+          name="cats"
+            >
+          Yes
+
+            <input type="radio"
+          name="cats"
+            >
+          No
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+          What is your favorite color?
+            <input type="color"
+          name="color"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+          What time do go to bed?
+            <input type="time"
+          name="time"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+          What are you favorite brand of cars?
+            <select name="brand"
+          multiple>
+              <option>BMW</option>
+              <option>Audi</option>
+              <option>Lada</option>
+            </select>
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+          How do you rate our work?
+            <input type="range"
+          name="rate"
+            >
+          </label>
+        </div>
+
+      </fieldset>
+
+      <fieldset class="set">
+        <legend>
+          Additional info:
+        </legend>
+
+        <div class="form-field">
+          <label>
+          Comments:
+            <textarea name="comments">
+
+            </textarea>
+          </label>
+        </div>
+
+        <div class="form-field">
+          <label>
+          Would you recommend us?
+            <select name="recommends">
+              <option>Yes</option>
+              <option>No</option>
+
+            </select>
+          </label>
+        </div>
+      </fieldset>
+
+      <button type="submit">Submit</button>
+
+
+
+    </form>
```

Comment:
Please fix code style/ Examle you can find in [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)
![image](https://user-images.githubusercontent.com/61904995/216343919-f445a81f-7b23-4e94-97b3-780fb2c5e7ee.png)